### ISSUE TITLE
All luner events start way too early.

### ISSUE THREAD

**Issue by MagmaSlime123** [Ellpeck/Nyx#44]
Title.

Forge 2854 (55 exists but FoamFix shits itself on 55), Nyx 1.4.0, MC 1.12.2.

![2021-05-27_17 07 41](https://user-images.githubusercontent.com/51277766/119907654-9a55fb80-bf05-11eb-9e0e-393e16f36137.png)

**Comment by democat3457** [comment]
what are your other mods @MagmaBro123

**Comment by MagmaSlime123** [comment]
FoamFix and BetterFoliage.

**Comment by democat3457** [comment]
does this happen on other worlds and/or with a reinstall?

**Comment by MagmaSlime123** [comment]
> does this happen on other worlds and/or with a reinstall?

Yes.

**Comment by democat3457** [comment]
two more questions: is this on the first day, or has time already progressed a lot? and secondly, and i hate to drive people away from the original mod, but does this happen with nyx rotn edition?

**Comment by MagmaSlime123** [comment]
> two more questions: is this on the first day, or has time already progressed a lot? and secondly, and i hate to drive people away from the original mod, but does this happen with nyx rotn edition?

Yes. Apparently (I figured this out after a bit of mod adding/removing shit) this is caused by (Both Bloodmoon and Nyx/Nyx RotN Edition are affected by this) Time4TFC, which is an addon mod for a mod known as TerraFirmaCraft, that changes the time of day/night depending on the time of the year.

**Comment by democat3457** [comment]
that addon sounds pretty hacky tbh, i can see how that would make a lot of time-dependent things go awry

**Comment by MagmaSlime123** [comment]
Well, it'd be nice to have some sort of compat. Talk to them about it, I'm none the wiser about its code. I'm not a dev.

**Comment by democat3457** [comment]
Did you bring up this issue with the Time4TFC devs? May have to be done on their side

https://github.com/AnodeCathode/Time4TFC/issues

**Comment by MagmaSlime123** [comment]
I doubt either mod is active, but will do.
Edit: Done. https://github.com/AnodeCathode/Time4TFC/issues/2#issue-914073766

I'll rephrase my issue here as well:

Luner events from both this mod and Bloodmoon don't sync with Time4TFC's day/night cycle changes.
